enterprise/coderd/prebuilds/metricscollector.go

```diff
@@ -82,6 +83,12 @@ var (
 		labels,
 		nil,
 	)
+	presetHardLimitedDesc = prometheus.NewDesc(
+		MetricPresetHardLimitedGauge,
+		"Indicates whether a given preset has reached the hard failure limit (1 for hard-limited, 0 otherwise).",
```

Comment:
Since now we only emit `1` and if a preset is no longer hard-limited, its entry is removed and no metric is emitted, meaning 0 is not actually reported, we should update this:
```suggestion
		"Indicates whether a given preset has reached the hard failure limit (1 = hard-limited). Metric is omitted otherwise.",
```